client/src/components/Utilities/GetCoordinates/GetCoordinates.tsx

```diff
@@ -254,13 +254,13 @@ const GetCoordinates: React.FC = () => {
             {genomicEnd != null ? renderRow("Genomic end", genomicEnd) : null}
             {renderRow("Transcript", results.tx_ac)}
             {txSegStart?.exon_ord != null
-              ? renderRow("Exon start", txSegStart.exon_ord)
+              ? renderRow("Exon start", txSegStart.exon_ord + 1)
```

Comment:
do we append 1 to the Tx Segment elements for these as well? If not but the behavior is different, we might want to take a look at if we should fix this in fusor instead

Reply:
So the JSON output represents the exon numbers using 0-indexing, but since the human-readable nomenclature string that populates above the Transcript Segment Element uses 1-indexing, I thought it would be good to use 1-indexing in this case since the convert coordinates output is also human readable